src/controllers/controller.line.js

```diff
@@ -148,6 +148,21 @@ module.exports = function(Chart) {
 			return borderWidth;
 		},
 
+		getPointRotation: function(point, index) {
+			var pointRotation = this.chart.options.elements.point.pointRotation;
+			var dataset = this.getDataset();
+			var custom = point.custom || {};
+
+			if (!isNaN(custom.pointRotation)) {
+				pointRotation = custom.pointRotation;
+			} else if (!isNaN(dataset.pointRotation) || helpers.isArray(dataset.pointRotation)) {
+				pointRotation = helpers.valueAtIndexOrDefault(dataset.pointRotation, index, pointRotation);
+			} else if (!isNaN(dataset.pointRotation)) {
```

Comment:
I don't think this case will ever be hit. If `!isNaN(dataset.pointRotation)` is true, the previous `else if` will run.

Reply:
Oops, right you are! Fixed.